src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -102,6 +103,17 @@ def auto_resolve_project_issues(project_id, cutoff=None, chunk_size=1000, **kwar
                 kwargs={"project_id": group.project_id, "group_id": group.id}
             )
 
+            analytics.record(
+                "issue.resolved",
```

Comment:
@scttcper damn, this is going to break a lot of analysis because we'll need to account for the resolution type. Maybe a separate event would be better?

Reply:
lets emit another event

Reply:
@scttcper Ok, if we do that let's change the other auto-resolve action